src/app/games/coerceo/coerceo.component.ts

```diff
@@ -56,10 +56,13 @@ export class CoerceoComponent extends TriangularGameComponent<CoerceoRules,
         this.state = this.getState();
         this.scores = MGPOptional.of(this.state.captures);
         this.tiles = this.state.tiles;
-        const move: MGPOptional<CoerceoMove> = this.node.move;
-        if (move.isPresent()) {
-            this.lastStart = move.get().start;
-            this.lastEnd = move.get().landingCoord;
+        const moveOpt: MGPOptional<CoerceoMove> = this.node.move;
+        if (moveOpt.isPresent()) {
+            const move: CoerceoMove = moveOpt.get();
+            if (move instanceof CoerceoNormalMove) {
+                this.lastStart = MGPOptional.of(move.getStart());
+                this.lastEnd = MGPOptional.of(move.getEnd());
+            }
```

Comment:
y'a pas de else, c'est voulu de laisser lastStart et lastEnd inchangé ?

Reply:
juste pour être sûr : on clear le last move avant d'appeler showLastMove ? Car si pas, on peut se retrouver avec le before-to-last move affiché, si le last move c'est un tile exchange. Si on clear bien avant, feel free to resolve !

Reply:
inderdaad, j'ai testé, c'est bien ce qu'il se passe.
Par contre on dit pas penultian en anglais ?

Reply:
penultimate
Si tu doutes d'un mot : https://www.wordreference.com/

Reply:
top, opr ?